Cargo.toml

```diff
@@ -16,7 +16,7 @@ members = [
 resolver = "2"
 
 [workspace.package]
-version = "0.3.0"
+version = "0.3.1"
```

Comment:
This was never merged from `release-0.3` back into `main`. Will move to another PR.